packages/shared/src/types/props/design.ts

```diff
@@ -108,6 +108,7 @@ type Visual = BannerDesignImage | ChoiceCardsDesign;
 
 export interface ConfigurableDesign {
     visual?: Visual;
+    headerImage?: BannerDesignImage;
```

Comment:
I'm probably worrying unnecessarily - but here we're saying the header image has a `kind` field but in SAC that was removed for the header image. Is it important that they match?

Reply:
Fixed - please check update